collect_app/src/test/java/org/odk/collect/android/application/initialization/SavepointsImporterTest.kt

```diff
@@ -26,17 +26,17 @@ class SavepointsImporterTest {
     private val component = DaggerUtils.getComponent(ApplicationProvider.getApplicationContext<Context>() as Application)
 
     private val projectsRepository = component.projectsRepository()
-    private val projectDependencyProviderFactory = component.projectDependencyProviderFactory()
+    private val projectDependencyProviderFactory = component.projectDependencyModuleFactory()
```

Comment:
`projectDependencyProviderFactory` variables should be renamed to `projectDependencyModuleFactory` as well.